### DISCUSSION TITLE
Donations

### DISCUSSION THREAD

**Discussion by morja** [Attendize/Attendize#1030] [Q&A] [answered]
Is there an option to receive an additional variable amount as donation on top of the ticket price?
And if not. Would it be easy to implement?

**Comment by johannac** [comment, marked as answer]
No there currently isn't an option to add a donation to an order. It's definitely possible to add something like that, but I think it would be a bit complicated to do, so might take a while to implement. 

Has anyone else implemented this or something similar in a fork?
